Library/Formula/aws-keychain.rb

```diff
@@ -0,0 +1,15 @@
+class AwsKeychain < Formula
+  desc "Uses OS X keychain for storage of AWS credentials."
+  homepage "https://github.com/pda/aws-keychain"
+  url "https://github.com/pda/aws-keychain/archive/v3.0.0.tar.gz"
+  sha256 "3c9882d3b516b629303ca9a045fc50f6eb75fda25cd2452f10c47eda205e051f"
+
+  def install
+    bin.install "aws-keychain"
+  end
+
+  test do
+    # It is not possible to create a new keychain without triggering a prompt.
```

Comment:
What sort of prompt? We can simulate basic interactivity with `IO.popen`; see [`datomic.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/datomic.rb) for an example.